engine/subsystem_types.go

```diff
@@ -37,6 +37,7 @@ var (
 	errNilExchangeManager           = errors.New("cannot start with nil exchange manager")
 	errNilDatabaseConnectionManager = errors.New("cannot start with nil database connection manager")
 	errNilConfig                    = errors.New("received nil config")
+	errAlreadyRunning               = errors.New("subsystem is already running")
```

Comment:
Why introduce this vs `ErrSubSystemAlreadyStarted`?

Reply:
Wasn't introduced, just moved. Looks like DataHistory manager used it as well, it has since been eliminated